subprojects/docs/src/snippets/tasks/inputNormalizationMetaInf/tests/runtimeClasspathNormalization.sample.conf

```diff
@@ -0,0 +1,2 @@
+executable: gradle
+args: tasks
```

Comment:
The snippet has the `java` plugin applied, maybe run `check` here instead?

Reply:
Agree, done